spec/features/lettings_log_spec.rb

```diff
@@ -110,15 +211,115 @@
-        visit("lettings-logs/#{log_id}/setup/check-answers")
-        expect(page).not_to have_content("Owning organisation #{user.organisation.name}")
-        expect(page).not_to have_content("Log owner #{user.name}")
+      context "and there is at most 1 potential stock owner" do
+        it "does not include the owning organisation and created by questions" do
+          visit("/lettings-logs")
+          click_button("Create a new lettings log")
+          click_link("Set up this lettings log")
+          log_id = page.current_path.scan(/\d/).join
+          expect(page).to have_current_path("/lettings-logs/#{log_id}/needs-type")
+          visit("lettings-logs/#{log_id}/setup/check-answers")
+          expect(page).not_to have_content("Stock owner ")
+          expect(page).not_to have_content("Log owner ")
+        end
+      end
+
+      context "and there are 2 or more potential stock owners" do
+        let!(:owning_org1) { create(:organisation, name: "Owning org 1") }
+        let!(:org_rel1) { create(:organisation_relationship, child_organisation: user.organisation, parent_organisation: owning_org1) }
+
+        it "does include the owning organisation question" do
+          visit("/lettings-logs")
+          click_button("Create a new lettings log")
+          click_link("Set up this lettings log")
+          log_id = page.current_path.scan(/\d/).join
+          expect(page).to have_current_path("/lettings-logs/#{log_id}/stock-owner")
+          visit("lettings-logs/#{log_id}/setup/check-answers")
+          expect(page).to have_content("Stock owner User org")
+        end
+
+        context "and there are 3 or more potential stock owners" do
+          let!(:owning_org2) { create(:organisation, name: "Owning org 2") }
+          let!(:org_rel2) { create(:organisation_relationship, child_organisation: user.organisation, parent_organisation: owning_org2) }
+
+          context "and the organisation relationship for the selected stock owner is deleted" do
+            it "doesn't change the CYA page text to be 'You didn't answer this question'" do
+              visit("/lettings-logs")
+              click_button("Create a new lettings log")
+              click_link("Set up this lettings log")
+              log_id = page.current_path.scan(/\d/).join
+              expect(page).to have_current_path("/lettings-logs/#{log_id}/stock-owner")
+              select(owning_org1.name, from: "lettings-log-owning-organisation-id-field")
+              click_button("Save and continue")
+              visit("lettings-logs/#{log_id}/setup/check-answers")
+              expect(page).to have_content("Stock owner Owning org 1")
+              org_rel1.destroy!
+              visit("lettings-logs/#{log_id}/setup/check-answers")
+              expect(page).to have_content("Stock owner Owning org 1")
+              expect(user.organisation.stock_owners).to eq(org_rel2.parent_organisation)
```

Comment:
Similarly to above, this line is pretty artificial/contrived way of explicitly referencing the organisation relationship in the test. Without it, Rubocop complains that something being defined with `let!` is not being explicitly used in the test. The reason I have defined the organisation relationship at all is because its existence influences the logic around the answer options for the question, and in particular whether the question gets auto-assigned an answer.